lib/themes/dosomething/paraneue_dosomething/includes/helpers.inc

```diff
@@ -361,11 +361,28 @@ function paraneue_dosomething_get_themed_reportback_item($data, $rogue_rb_item =
     return NULL;
   }
 
-  if ($data->id && !dosomething_kudos_disable_reactions($data->campaign->id) && !$rogue_rb_item) {
-    $data->kudos = dosomething_kudos_get_kudos_data_for_fid($data->id);
-  }
+  if ($data->id && !dosomething_kudos_disable_reactions($data->campaign->id)) {
+      if ($rogue_rb_item) {
+        // Find the fid from the rogue reference table
+        // $data->id = db_query("SELECT rogue_rbs.fid FROM {dosomething_rogue_reportbacks} rogue_rbs WHERE rogue_event_id = :rogue_event_id", array(':rogue_event_id' => $data->event_id))->fetchField();
+
+        $total_rogue_kudos = $data->kudos['total'];
+        $liked_by_current_user = $data->kudos['liked_by_current_user'] ? true : false;
+
+        if ($liked_by_current_user) {
+          drupal_add_js(
+            [
+              'dsKudosReactions' => [
+                'likedByCurrentUser' => $liked_by_current_user,
+              ]
```

Comment:
@DFurnes adding this setting to pass data to the front end if a user has liked the current reportback item. Do I need to add another level per item so it doesn't make this value be the same for each reportback item when it sends to the front end? Maybe this isn't the best way to do this...